plugin/r/src/main/java/com/twosigma/beaker/r/utils/RServerEvaluator.java

```diff
@@ -108,9 +110,8 @@ public void evaluate(SimpleEvaluationObject seo, String code) {
   }
 
   public List<String> autocomplete(String code, int caretPosition) {
-    logger.fine("not implemented");
-    // TODO
-    return null;
+    logger.fine("autocomplete");
```

Comment:
Yea, I added logging to the Java modules I recently updated. You can enable it by setting 'beaker_logger_file' environment variable to point to a Java logger configuration. If you don't do it then everything is disabled.